### DISCUSSION TITLE
Output board question on C12

### DISCUSSION THREAD

**Discussion by electro991** [CrashOverride85/zc95#115] [Q&A] [answered]
Hi again, 

I have had some issues building my 2nd unit. When I did initial start up, I had a calibration failure on the output board. After a few seconds while looking at the debug output, I saw that C12 was smoking. I am thinking it may be one of those doa component you get from time to time but figured I'd check here first, before I order and replace C12. Assembly of smd was completed by JLC pcb.

Voltage recorded on debug was very low, like 0.012 or something.

Can you think of anything that could cause C12 to smoke other than C12 itself ?

I build a 3rd output board and put in in place and it worked first boot so eveything else about the unit seems correct.

**Comment by CrashOverride85** [comment, marked as answer]
That cap is between the 9v rail and ground, the only reasons I can possibly think of for it to smoke is if it either got installed backwards, or was faulty/damaged. The low voltage reported makes sense - for the cap to start smoking, it's probably close to a dead short, so the 9v supply would be closer to 0...
BTW the line on tantalum caps is +, not - as with other types

**Reply by electro991** [reply]
Thank you. Indeed damaged, I tested TP1 to ground and it showed short. Removed C12 from PCB and no longer short. I will order a replacement part and re-install a new cap.

**Reply by CrashOverride85** [reply]
If you haven't tried already, the board should work well enough to pass the self test/calibration without that cap, so long as the others are good. Whilst located next to each channels output stage, C12/13/14/15 are in parallel

**Reply by electro991** [reply]
Will do, waiting on a pico to test

**Reply by electro991** [reply]
Fixed. The board now passes self test. I have replaced C12. Lesson is that JLCPCB assembled a faulty cap on that board.
